### PR TITLE
Flag empty airtable rows on data-validation page

### LINKED ISSUE #1087: Flag empty airtable rows on data-validation page

### PR DESCRIPTION
**Pull request by smcmurtry**
Closes #1087

### CODE REVIEW

File: pages/data-validation.js [hunk 1]
```diff
@@ -94,6 +94,8 @@ export class DataValidation extends Component {
     }
   }
 
+  checkForTablesWithEmptyRows() {}
```
Comment (sastels) [inline pages/data-validation.js]: what is this empty function for?

File: utils/airtable_es2015.js [hunk 2]
```diff
@@ -42,14 +42,19 @@ var hydrateFromAirtable = (exports.hydrateFromAirtable = async function hydrateF
...
+  dataStore["errors"] = [];
   airtableConstants.tableNames.forEach(function(tableName) {
     var array = dataStore[tableName].map(x => Object.keys(x).length);
     var number_of_fields = Math.max(...array);
     dataStore[tableName] = dataStore[tableName].filter(x => {
       var fraction_of_cols_filled =
         (Object.keys(x).length * 1) / number_of_fields;
-      return fraction_of_cols_filled < 0.5 ? false : true;
+      if (fraction_of_cols_filled < 0.5) {
+        dataStore["errors"].push("missingValues." + tableName);
```
Comment (sastels) [inline utils/airtable_es2015.js]: If there are multiple rows with missing values then this message will be duplicated in `errors` - is that the intent?

**Review by sastels — APPROVED** [review]
minor comments, LGTM

File: pages/data-validation.js (hunk 1 above)
Comment (smcmurtry) [reply pages/data-validation.js]: whoops!

File: utils/airtable_es2015.js (hunk 2 above)
Comment (smcmurtry) [reply utils/airtable_es2015.js]: Good point, I just added the row number to the error message

**Pull request merged**